src/apps/backend/modules/task/internal/store/task-db.ts

```diff
@@ -20,7 +21,12 @@ export const TaskDbSchema: Schema = new Schema<TaskDB>(
       index: true,
       required: true,
     },
-    name: {
+    description: {
+      type: String,
+      index: true,
```

Comment:
`index: true` is not required if we do not have any search by description reader method.